tasks/mvp_todos.md

```
@@ -32,11 +32,12 @@
 - [ ] Implement LLM summarization for new IssueClusters @Sam @Backend @LLM
 - [x] Store cluster embedding_centroid @Colleague @Frontend
 
+### Coding Agent (External AWS Service)
 ### Coding Agent (External AWS Service)
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)